include/pybind11/attr.h

```diff
@@ -8,9 +8,36 @@
     BSD-style license that can be found in the LICENSE file.
 */
 
-#pragma once
+#include <Python.h>
+#include <stdint.h>
+#include <string.h>
+
+#include <string>
+#include <type_traits>
+#include <typeinfo>
+#include <utility>
+#include <vector>
+
+#include "pybind11/cast.h"
+#include "pybind11/chrono.h"
+#include "pybind11/complex.h"
+#include "pybind11/detail/class.h"
+#include "pybind11/detail/common.h"
+#include "pybind11/detail/init.h"
+#include "pybind11/detail/internals.h"
+#include "pybind11/detail/typeid.h"
+#include "pybind11/embed.h"
+#include "pybind11/eval.h"
+#include "pybind11/functional.h"
+#include "pybind11/iostream.h"
+#include "pybind11/numpy.h"
+#include "pybind11/operators.h"
+#include "pybind11/pybind11.h"
+#include "pybind11/pytypes.h"
+#include "pybind11/stl.h"
+#include "pybind11/stl_bind.h"
 
-#include "cast.h"
```

Comment:
I'd undo this change, I believe.

Reply:
It put in longer paths (`third_party/pybind11/include/pybind11`), I used `sed` to cut out the prefix but decided to keep the `pybind11` (having "explicit is better than implicit" in my head). I could cut out `pybind11` as well. BUT, I think the main question is, is the list of includes it produces useful, for cross-checking and/or occasional manual fixing up?

From this quick experiment I feel the tool is a lot too far away from what we'd need for fully automatic processing.